packages/vite/src/node/plugins/css.ts

```diff
@@ -1780,7 +1860,7 @@ async function minifyCSS(
       ...config.css?.lightningcss,
       targets: convertTargets(config.build.cssTarget),
       cssModules: undefined,
-      filename: cssBundleName,
+      filename: filename || cssBundleName,
       code: Buffer.from(css),
       minify: true,
     })
```

Comment:
I noticed that the same thing should be done for lightningcss warnings too.

Reply:
I added the logic to for the lightningcss, too. However, the warning itself just prints the message at the moment and the location doesn't seem to have any meaning. I don't see any api of lightningcss like `formatMessage()` from esbuild. What should I do here?

Reply:
I'll put it like this for now.

```ts
config.logger.warn(
  colors.yellow(
    `warnings when minifying css:\n${msgs
      .map(
        (w) =>
          `${w.message}\n\n${w.loc.filename}:${w.loc.line}:${w.loc.column}\n\n`,
      )
      .join('\n')}`,
  ),
)
```